src/Spice86.Core/Emulator/CPU/VectorTableManager.cs

```diff
@@ -0,0 +1,38 @@
+namespace Spice86.Core.Emulator.CPU;
+
+using Spice86.Core.Emulator.Memory;
+
+/// <summary>
+/// Manages accesses to the vector table
```

Comment:
Represents the Vector Table

Reply:
It's not the actual table, just a wrapper. Will rework the class to make that clearer

Reply:
Rewrote the class with an indexer, much cleaner.